src/stringBuffer.js

```diff
@@ -29,7 +29,17 @@
  * @return {function}
  */
 function makeBuffer() {
-  // write code here
+  let stringOfBuffer = '';
+  let getEmptyString = function() {
+    stringOfBuffer = '';
+  };
+  return function resultOfBuffer(value) {
+    if (value === undefined) {
+      return stringOfBuffer;
+    }
+    stringOfBuffer += value;
+    resultOfBuffer.clear = getEmptyString;
```

Comment:
moved body getEmptyString outside of resultOfBuffer

Reply:
No, a comment was not about it.
Move assignment to the function object (`resultOfBuffer.clear =`) from function because every time you call buffer it will be assigned

Reply:
I do not understand how can i do it?